code/game/objects/items/stacks/medical.dm

```diff
@@ -147,6 +147,27 @@
 	stack_id = "Tear petals"
 
 
+/obj/item/stack/medical/advanced
+	dir = NORTH
+	flags_atom = DIRLOCK
+
+/obj/item/stack/medical/advanced/update_icon()
+	if(max_amount < 1)
+		return
+	var/percentage = round(amount / max_amount) * 100
+	if(max_amount >= amount)
```

Comment:
Combine it with the first check for that double sweet early return.

Reply:
That would be something like `if(max_amount <1 || max_amount < amount)` for an early return.

Reply:
changed